__tests__/app.test.js

```diff
@@ -115,3 +117,43 @@ describe("Error Handling 404 ", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("return 200 status, should get a comment getby its article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+
+        expect(article).toHaveProperty("comment_id");
```

Comment:
It would be ideal to assert that each property has a value attached to it as well rather than just keys :)